marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/hub-entities/generateSearchOptions.sjs

```diff
@@ -359,6 +359,31 @@ function testHubCentralSupportedDatatypeMappingsForSort() {
   return assertions;
 }
 
+function generateExplorerOptionsWithoutContainerConstraint() {
+  const input =
+    [{
+      "info": {
+        "title": "Book"
+      },
+      "definitions": {
+        "Book": {
+          "properties": {
+            "publishedAtAddress": {"$ref": "#/definitions/Address"}
+          }
+        },
+        "Address": {
+          "properties": {}
+        }
+      }
+    }];
+  const options = hent.dumpSearchOptions(input, true);
+
+  return [
+    test.assertNotExists(options.xpath("/*:constraint[@name = 'Book']/*:container")),
```

Comment:
These assertions should have a message explaining why we don't want these here - i.e. touch on the Collection constraint and the fact that Explorer doesn't need these container constraints.